src/main/java/calculator/RunCalculator.java

```diff
@@ -0,0 +1,147 @@
+package calculator;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * 계산기
+ */
+public class RunCalculator {
+
+    /**
+     * 계산 시작
+     */
+    public static int runCalculator(final String paramText){
+        paramTextCheck(paramText);
+        String refineText = paramText.replaceAll(" ", "");
+        List<OperatorOrder> list = operatorDivision(refineText);
+        int returnValue = stepToCalculatorRun(refineText, list);
+
+        return returnValue;
+    }
+
+    /**
+     * 계산에 사용될 Text를 Check
+     * @param paramText
+     * @return 정제된 문자열
+     */
+    public static void paramTextCheck(final String paramText){
```

Comment:
해당 메서드는 RunCalculator 내부에서만 사용되고 있기 때문에 외부에 공개할 필요가 없어보이는데 어떻게 생각하시나요 :) ?

Reply:
메서드는 가능하면 동사구로 시작해주세요!

```suggestion
    public static void checkParamText(final String paramText) {
```